reference/compiling_for_android.rst

```diff
@@ -215,14 +198,14 @@ failure should be presented there.
 
 Seek assistance if you can't figure it out.
 
-Application exits immediately
+2.) Application exits immediately
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If the application runs but exits immediately, there might be one of the
 following reasons:
 
--  libgodot_android.so is not in ``libs/armeabi``
+-  libgodot_android.so is not in ``lib/armeabi-v7a``
```

Comment:
Then it's a bug in the Windows buildsystem. For the official binaries built on OSX: https://github.com/GodotBuilder/godot-builds/blob/master/.travis.yml#L161